morphct/chromophores.py

```diff
@@ -159,18 +159,16 @@ def __repr__(self):
         )
 
     def _set_center(self, snap, atom_ids):
-        box = snap.configuration.box[:3]
-        unwrapped_pos = snap.particles.position + snap.particles.image * box
-        center = np.mean(unwrapped_pos[atom_ids], axis=0)
-        img = np.zeros(3)
-        while (center + img * box < -box / 2).any() or (
-            center + img * box > box / 2
-        ).any():
-            img[np.where(center < -box / 2)] += 1
-            img[np.where(center > box / 2)] -= 1
+
+        box_dims = snap.configuration.box[:3]
+        unwrapped_pos = snap.particles.position[atom_ids] + snap.particles.image[atom_ids] * box_dims
+        center = np.mean(unwrapped_pos, axis = 0)
+        box = freud.Box.from_box(box_dims)
```

Comment:
We could change this to use freud's unwrap function which would handle non-orthorhombic boxes as well:
```suggestion
        box = freud.Box.from_box(snap.configuration.box)
        unwrapped_pos = box.unwrap(snap.particles.position[atom_ids], snap.particles.image[atom_ids])
        center = np.mean(unwrapped_pos, axis = 0)
```